packages/git_restore_mtime.rb

```diff
@@ -0,0 +1,34 @@
+# Adapted from Arch Linux git-tools-git PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=git-tools-git
+
+require 'package'
+
+class Git_restore_mtime < Package
```

Comment:
Isn't there an existing gittools package?

Reply:
Yes that's my point with a different name. But the existing package in Chromebrew is entirely different.

Reply:
I just checked and that is a different repository.  That said, since this includes more executables than just git-restore-mtime, would it make more sense to give it a more generic name like maybe git_utils?

Reply:
I was considering that, but thought that the best of mediocre solutions would just be to replicate a major upstream package name, so that people could easily find it if they were looking.

See https://packages.debian.org/sid/git-restore-mtime

Reply:
Actually, looking at that, how about `git-mestrelion-tools`?

Reply:
Not sure what mestrelion means but I guess it would be fine.